erin_averill/forLoop_basicII.py

```diff
@@ -0,0 +1,59 @@
+# # Biggie Size - Given an array, write a function that changes all positive numbers in the array to "big". 
+# # Example: makeItBig([-1, 3, 5, -5]) returns that same array, changed to [-1, "big", "big", -5].
+
+# def biggie(arr):
+# 	for i in arr:
+# 		if arr[i] > 0:
+# 			arr[i] = 'big'
+# 	return arr
+
+# biggie([-1,3,5,-5])
+# # Count Positives - Given an array of numbers, create a function to replace last value with number of positive values. 
+# # Example, countPositives([-1,1,1,1]) changes array to [-1,1,1,3] and returns it.  (Note that zero is not considered to be 
+# # a positive number).
+
+def countPositions(arr):
+	sum = 0
+	for i in arr:
+		if arr[i] < 0:
```

Comment:
in the for loop, i is each element in the array. 1st iteration gives arr[-1] //3